_posts/2018-10-16-newpipe-0.14.2-released.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: NewPipe 0.14.2 released, introducing grid layout for tablets
+short: "NewPipe 0.14.2 released"
+date: 2018-10-16 13:00:00 +0200
+categories: [release, pinned]
+author: TobiGr
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+We have just released NewPipe 0.14.2. Such as NewPipe 0.14.0 this version mostly brings UI and UX improvements
+which include new layouts for tablets and an additional linear layout in "Downloads".
+
+Furthermore, the subscriptions page got a small upgrade, several bugs were fixed and new translations added.
```

Comment:
Try to avoid "get" if possible. I'd further split the sentence into two parts.

```suggestion
Furthermore, the subscriptions page receievd a small upgrade. Also, several bugs were fixed and new translations were added.
```